src/main/java/executor/service/config/bean/ThreadPoolExecutor.java

```diff
@@ -32,7 +35,7 @@ public ExecutorService threadPoolExecutor() {
         ThreadPoolConfig threadPoolConfig = threadPoolConfig();
         var properties = propertiesConfig.getProperties(THREAD_POOL_PROPERTIES);
         int size = Integer.parseInt(properties.getProperty(MAXIMUM_POOL_SIZE));
-        return new ThreadPoolExecutor(
+        return new java.util.concurrent.ThreadPoolExecutor(
```

Comment:
return new java.util.concurrent.ThreadPoolExecutor and public class ThreadPoolExecutor, what sense of this actions?

Reply:
It was agreed.